src/test/java/seedu/docit/logic/commands/AppointmentCommandTestUtil.java

```diff
@@ -8,21 +8,22 @@
 import seedu.docit.commons.core.index.Index;
 import seedu.docit.model.Model;
 import seedu.docit.model.appointment.Appointment;
-import seedu.docit.model.appointment.AppointmentContainsKeywordsPredicate;
+import seedu.docit.model.appointment.AppointmentContainsPatientPredicate;
+import seedu.docit.model.patient.Patient;
 
 public class AppointmentCommandTestUtil {
 
     /**
-     * Updates {@code model}'s filtered list to show only the appointment at the given {@code targetIndex} in the
-     * {@code model}'s appointment list.
+     * Updates {@code model}'s filtered list to show appointments with patient at the given {@code targetIndex} in the
+     * {@code model}'s patient list.
```

Comment:
Should it be appointment list here?

Reply:
Since the original problem was that appointment references a Patient based on the Patient's index in the addressbook, I have modified the `showAppointmentWithPatientAtIndex` method to show appointments with the **Patient at its addressbook index** instead.

Reply:
Ah, I see thanks for clarifying